base/applications/regedit/lang/ro-RO.rc

```diff
@@ -442,28 +442,28 @@ BEGIN
     ID_EDIT_NEW_STRINGVALUE "Adaugă o nouă valoare șir"
     ID_EDIT_NEW_BINARYVALUE "Adaugă o nouă valoare de 1 bit"
     ID_EDIT_NEW_DWORDVALUE "Adaugă o nouă valoare de 32 de biți"
-    ID_REGISTRY_IMPORTREGISTRYFILE "Importă registru dintr-un fișier"
+    ID_REGISTRY_IMPORTREGISTRYFILE "Importă regiștri dintr-un fișier"
     ID_REGISTRY_EXPORTREGISTRYFILE "Exportă registrul total sau parțial într-un fișier"
-    ID_REGISTRY_LOADHIVE "Încarcă un fișier binar în registru"
-    ID_REGISTRY_UNLOADHIVE "Eliberează conținutul unui fișier binar din registru"
-    ID_REGISTRY_CONNECTNETWORKREGISTRY "Conectare la registrul unui calculator din rețea"
-    ID_REGISTRY_DISCONNECTNETWORKREGISTRY "Deconectare de la registrul calculatorului din rețea"
-    ID_REGISTRY_PRINT "Imprimă toate părțile unui registru"
-/*    ID_HELP_HELPTOPICS "Deschide manualul editorului de registru" */
+    ID_REGISTRY_LOADHIVE "Încarcă un fișier binar în regiștri"
+    ID_REGISTRY_UNLOADHIVE "Eliberează conținutul unui fișier binar din regiștri"
+    ID_REGISTRY_CONNECTNETWORKREGISTRY "Conectare la regiștrii unui calculator din rețea"
+    ID_REGISTRY_DISCONNECTNETWORKREGISTRY "Deconectare de la regiștrii calculatorului din rețea"
+    ID_REGISTRY_PRINT "Imprimă tot sau o parte din regiștri"
+/*    ID_HELP_HELPTOPICS "Deschide termenii de ajutor a editorului de regiștri" */
```

Comment:
Do not touch this line, you do create a merge issue with an existing PR.

Reply:
I did resolve the conflicts for you. Please do git pull from that branch before continuing your work on top of it.

Reply:
currently resolved.